### DISCUSSION TITLE
Add {macos,ubuntu,windows}-penulimate label

### DISCUSSION THREAD

**Discussion by jendrikseipp** [actions/virtual-environments#6119] [Ideas]
For people like me who'd like to always test the latest two OS versions, it would be great to have `{macos,ubuntu,windows}-penulimate` labels in addition to `{macos,ubuntu,windows}-latest`.

**Comment by mikemahoney218** [comment]
+1

In addition, I think it's a semi-common use-case to test on older OS versions, to ensure maximum compatibility; I'd love if there were a floating tag `oldest` that always pointed at the oldest-supported version of a given OS.

**Comment by mikhailkoliada** [comment]
Hello!

We do not currently have plans to introduce more labels I am afraid, so the only way to check what is penultimate is still to track the runner-images repo itself.
